src/views/DatasetManager/components/DatasetOverview/components/CategoryAccordion/components/KPI/KPI.js

```diff
@@ -34,7 +35,11 @@ const KPI = (props) => {
       <Grid item>{label}</Grid>
 
       <Grid item>
-        <KPIValue kpi={kpi} valueTypographyProps={{ ...valueProps }} />
+        {kpi.state === KPI_STATE.IDLE || kpi.state === KPI_STATE.LOADING ? (
+          <Skeleton width="24px" />
+        ) : (
+          <KPIValue kpi={kpi} valueTypographyProps={{ ...valueProps }} />
+        )}
```

Comment:
Minor change suggestion: instead of implementing this condition in the `KPI` component, we can implement this logic in the `KPIValue` component, and revert the changes in this file. The skeleton would then be returned by `KPIValue` when the state is `IDLE` or `LOADING` (otherwise the current behavior returns the same result as the "UNKNOWN" state).

I'll add a separated _refactor_ commit for this change and merge the branch, so we can still discuss and revert it later if we prefer to keep the current version :slightly_smiling_face: